mitodl/ocw-studio
# Static_Shared Historical artifacts removed

#### Pre-Flight checklist


#### What are the relevant tickets?
#1681 

#### What's this PR do?
It removes all the historical webpack assets when a course is published

#### How should this be manually tested?
- Checkout to this branch
- Make sure docker containers are running
- Run `docker-compose exec web ./manage.py upsert_theme_assets_pipeline`
- Go to concourse
- Run Theme-Assets pipeline
- Once finished, publish a course (Staging/ Draft)
- Go to minio, ocw-content-draft/courses/<published-course>/static_shared/css
- You should see single instances of webpack assets
   - www
   - course-v2
   - fields
